packages/cli-kit/src/private/node/ui/components/Prompt/Prompt.test.tsx

```diff
@@ -1,5 +1,5 @@
 import Prompt from './Prompt.js'
-import {sendInput, waitForInputsToBeReady} from '../../../../testing/ui.js'
```

Comment:
I noticed that, since our last discussion about "your" `Table` component, "my" `Table` is now in a separate folder, so, if you don't mind, I keep "your" `Table`’s name unchanged, and leave the remaining work to you :)